docs/codeql/codeql-cli/publishing-and-using-codeql-packs.rst

```diff
@@ -53,22 +55,24 @@ To run a pack that someone else has created, you must download it by running the
 - ``<pack>``: the name for the pack that you are creating.
 - ``@x.x.x``: an optional version number. If omitted, the latest version will be downloaded.
 
+This command accepts arguments for multiple packs, as well as query suites.
```

Comment:
Hmmmm...I think you should either remove the mentioning of "query suites" or go into more detail on what it means to pass that as an argument. In the short term, we might be able to consider passing a query suite as an _advanced_ or _wizards only_ option. So, we can avoid mentioning it here and simplify the docs.

Otherwise, you will need to go into some detail on how we analyze query suites for dependent packs.

Reply:
I had been ignoring query suites because I was fairly confident they weren't relevant for 2.5.8. 

If query suites are something we want to encourage, but they'll need more explanation, maybe we should add this to the docs in a follow up issue?

Reply:
Yes, let's ignore them for now.